src/scrapers/main.py

```diff
@@ -1,16 +1,19 @@
 import logging
 from optparse import OptionParser
 
+from models import InitiativeGroup
 from platformen import NLvoorElkaar, WijAmsterdam, CoronaHelpersScraper
-    # HeldNodig, MensenDieWillenHelpen, Zorgheldenauto, PuurPapendrecht, NijmegenOost
+# HeldNodig, MensenDieWillenHelpen, Zorgheldenauto, PuurPapendrecht, NijmegenOost
 from tools import Geocoder
 
 logging.basicConfig(level=logging.DEBUG)
 
 parser = OptionParser(add_help_option=False)
 parser.add_option("-h", "--help", help="show this help message and exit", action="store_true")
 parser.add_option("-l", "--limit", help="LIM is the amount of items each scraper scrapes.", type="int", metavar="LIM")
-parser.add_option("-g", "--nogeo", help="Disables the geocoder", action="store_true", dest="nogeo")
+parser.add_option("-g", "--group=SIDE", help="Constrain supporting scrapers to only one SIDE: 'supply' or 'demand'",
```

Comment:
werkt dit?
```
group_parser = parser.add_mutually_exclusive_group(help="Constrain supporting scrapers to only one SIDE: 'supply' or 'demand'", dest="group")
group_parser.add_argument(InitiativeGroup.SUPPLY)
group_parser.add_argument(InitiativeGroup.DEMAND)
```

Reply:
Oh, misschien is dit beter
`parser.add_argument("-g", "--group", choices=[InitiativeGroup.SUPPLY, InitiativeGroup.DEMAND], help="Constrain supporting scrapers to only one SIDE: 'supply' or 'demand'", dest="group")`